src/openrct2/scenario/ScenarioRepository.cpp

```diff
@@ -150,9 +151,11 @@ class ScenarioRepository final : public IScenarioRepository
         std::string rct1dir = _env->GetDirectoryPath(DIRBASE::RCT1, DIRID::SCENARIO);
         std::string rct2dir = _env->GetDirectoryPath(DIRBASE::RCT2, DIRID::SCENARIO);
         std::string openrct2dir = _env->GetDirectoryPath(DIRBASE::USER, DIRID::SCENARIO);
+        std::string rct1datdir = _env->GetDirectoryPath(DIRBASE::RCT1, DIRID::DATA);
         Scan(rct1dir);
         Scan(rct2dir);
         Scan(openrct2dir);
+        ScanMegaPark(rct1datdir, rct1dir);
```

Comment:
My OpenRCT2 directory didn't have a scenario folder by default, so it threw an exception. It works once you add a scenario folder, but shouldn't it do that by default?